hack/test-update-storage-objects.sh

```diff
@@ -176,8 +175,7 @@ killApiServer
 
 KUBE_API_VERSIONS="${KUBE_NEW_API_VERSION},${KUBE_OLD_API_VERSION}"
 RUNTIME_CONFIG="api/all=false,api/${KUBE_OLD_API_VERSION}=true,api/${KUBE_NEW_API_VERSION}=true"
-# TODO: Switch to STORAGE_BACKEND_ETCD3 once we support it.
-startApiServer ${STORAGE_BACKEND_ETCD2} ${KUBE_NEW_STORAGE_VERSIONS} ${KUBE_STORAGE_MEDIA_TYPE_JSON}
+startApiServer ${STORAGE_BACKEND_ETCD3} ${KUBE_NEW_STORAGE_VERSIONS} ${KUBE_STORAGE_MEDIA_TYPE_JSON}
```

Comment:
Currently, this is just the first step of test, because we are using 3.0.10 binary in all runs. So we are actually only testing migration from v2 to v3 API, not upgrading the binary itself. I will try to improve the test later (once we have the "migrate-if-needed.sh" script fully ready.